### PR TITLE
KTL-410: og images

### PR DESCRIPTION
**Pull request by zoobestik**

### CODE REVIEW

**Review by nikpachoo — COMMENTED** [review]
The image "assets/images/open-graph/docs-mobile.png" seems a little bit distorted 😊

**Review by nikpachoo — COMMENTED** [review]
I propose moving "play.png" to the play repo to avoid the implicit connection between projects.

**Review by nikpachoo — APPROVED** [review]

**Review by krutilov — APPROVED** [review]

File: docs/cfg/buildprofiles.xml [hunk 1]
```diff
@@ -10,7 +10,9 @@
         </browser-edits-url-override>
         <kotlin-latest-url>%kotlinLatestUrl%</kotlin-latest-url>
         <generate-only-default-anchors>true</generate-only-default-anchors>
-        <web-root>https://www.jetbrains.com/idea/</web-root>
+        <web-root>https://kotlinlang.org/</web-root>
```
Comment (koshachy) [inline docs/cfg/buildprofiles.xml:13]: I guess there should be `https://kotlinlang.org/docs/` ?

Comment (koshachy) [reply docs/cfg/buildprofiles.xml:13]: According to the WH team announcement:

> web-root— path to help instance, everything that precedes product web-path. For example, all instances hosted on jetbrains.com must specify web-root as https://www.jetbrains.com/help

Comment (zoobestik) [reply docs/cfg/buildprofiles.xml]: resolved

**Pull request merged**